data-integration/main.bal

```diff
@@ -0,0 +1,103 @@
+import ballerina/http;
+import ballerina/os;
+import ballerina/persist;
+import ballerina/lang.runtime;
+import data_integration.db;
```

Comment:
The convention is to add imports from the same package before `ballerina/` imports.